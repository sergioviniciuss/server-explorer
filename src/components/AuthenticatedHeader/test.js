import React from "react";
import { render } from "@testing-library/react";
import { AuthenticatedHeader } from ".";

describe("AuthenticatedHeader", () => {
  it("renders the component properly", () => {
    const { container } = render(<AuthenticatedHeader />);

    expect(container.firstChild).toMatchSnapshot();
  });

  it("renders logo in the header", () => {
    const { getByRole } = render(<AuthenticatedHeader />);

    expect(getByRole("img")).toBeInTheDocument();
  });

  it("renders greeting message in the header", () => {
    const { getByText } = render(<AuthenticatedHeader />);

    expect(getByText(/hello there!/i)).toBeInTheDocument();
  });
});
