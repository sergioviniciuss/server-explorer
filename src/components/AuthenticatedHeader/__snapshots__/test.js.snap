// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthenticatedHeader renders the component properly 1`] = `
<div
  class="headerContainer"
>
  <img
    class="logo"
    src="/file"
  />
  <span
    class="greetings"
  >
    Hello there!
  </span>
</div>
`;
